sources/packages/api/src/services/application/application.service.ts

```diff
@@ -67,10 +69,10 @@ export const INSUFFICIENT_APPLICATION_SEARCH_PARAMS =
 export class ApplicationService extends RecordDataModelService<Application> {
   @InjectLogger()
   logger: LoggerService;
-  private bypassApplicationSubmitValidations: boolean;
-
+  private readonly config: IConfig;
   constructor(
     connection: Connection,
+    private readonly configService: ConfigService,
```

Comment:
As per my previous comment, configService instance does not need to be global variable.